### PR TITLE
DEVLIB-1257: Rename Compute@Edge to Compute

### PR DESCRIPTION
**Pull request by doramatadora**
JIRA ticket: [DEVLIB-1257](https://fastly.atlassian.net/browse/DEVLIB-1257)

Please review and notify DevRel (and @Integralist) of the new Viceroy release 🙌🏼  

[DEVLIB-1257]: https://fastly.atlassian.net/browse/DEVLIB-1257?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kpfleming — APPROVED** [review]
Once the style guide has been published we'll have to revisit some of this, but it's not worth changing right now.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Viceroy
 
-Viceroy provides local testing for developers working with Compute@Edge. It
-allows you to run services written against the Compute@Edge APIs on your local
+Viceroy provides local testing for developers working with Compute. It
```
Comment (ulyssa) [inline README.md:3]: I think that this would read clearer if the first mention was "Fastly Compute" instead.

File: cli/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "viceroy"
-description = "Viceroy is a local testing daemon for Compute@Edge."
+description = "Viceroy is a local testing daemon for Compute."
```
Comment (ulyssa) [inline cli/Cargo.toml:3]: I think the package description should probably also say Fastly Compute.

File: README.md (hunk 1 above)
Comment (doramatadora) [inline README.md:3]:
```suggestion
Viceroy provides local testing for developers working with Fastly Compute. It
```

File: lib/compute-at-edge-abi/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # 🔗 compute-at-edge-abi
```
Comment (JakeChampion) [inline lib/compute-at-edge-abi/README.md:1]: @dora and @kpfleming - do we want to rename this and the folder?

Comment (kpfleming) [reply lib/compute-at-edge-abi/README.md:1]: Oh... good question. That's going to be a breaking change for anything which consumes the content here?

Comment (kpfleming) [reply lib/compute-at-edge-abi/README.md:1]: Maybe not.. it appears that the WITX files are incorporated into the crate but the directory name is not part of the crate.

Comment (kpfleming) [reply lib/compute-at-edge-abi/README.md:1]: But one of the WITX files is named `compute-at-edge.witx`.

**Pull request merged**